lib/miq_automation_engine/models/miq_ae_field.rb

```diff
@@ -80,12 +80,7 @@ def to_export_xml(options = {})
   end
 
   def substitute=(value)
-    # Any invalid boolean string should be converted to default
-    column_class = self.class.columns_hash['substitute'].class
-    unless column_class::TRUE_VALUES.include?(value) ||
-           column_class::FALSE_VALUES.include?(value)
-      value = DEFAULTS[:substitute]
-    end
+    value = false if value.nil?
```

Comment:
is substitute a boolean column?  Can it just stay as nil so we don't have to override?  cc @mkanoor